server/routes/reviewsHelpers.js

```diff
@@ -122,18 +125,74 @@ const addHelpfulVote = (reviewId) => {
       return data.status
     })
     .catch((err) => {
-      console.error('ERROR in addHelpfulVote: ', err);
+      console.error('ERROR in addHelpfulVote: ', err)
     });
 }
 
+const formatPostData = (postData) => {
+  console.log('TWO')
+  return new Promise ((resolve, reject) => {
+    let formattedPostData = {};
+
+    formattedPostData['product_id'] = Number(postData.product_id);
```

Comment:
Good work here, but just a suggestion. This could be a great place to practice object deconstruction if you ever find free time to refactor later on.